### DISCUSSION TITLE
moment() in frontmatter

### DISCUSSION THREAD

**Discussion by huenchen** [SilentVoid13/Templater#1335] [Help]
Since last week I cannot use a function I had been using a long time. In the frontmatter I used:
`week: <%moment(tp.file.title).format("[W]ww")%>`
I cannot even remove it in the parsed file. Is it still possible to use this function in the frontmatter?
Thanks!

**Comment by Zachatoo** [comment]
I can use that code in frontmatter. Are there any errors in the [dev tools console](https://help.obsidian.md/Help+and+support#Capture+console+logs) when you try to apply the template? If you disable all other community plugins and restart Obsidian does the issue still occur?

Here's the template I tested with.
```js
---
key: <%moment(tp.file.title).format("[W]ww")%>
---
```

Here's my [debug information](https://help.obsidian.md/Help+and+support#Capture+console+logs) on the device I tested on.
```
SYSTEM INFO:
	Operating system: android 14 (Google Pixel 7a)
	Obsidian version: 1.5.8 (126)
	API version: v1.5.8
	Login status: not logged in
	Live preview: on
	Base theme: dark
	Community theme: none
	Snippets enabled: 17
	Restricted mode: off
	Plugins installed: 20
	Plugins enabled: 19
		1: Iconize v2.7.3
		2: Hotkey Helper v0.3.18
		3: Advanced URI v1.40.0
		4: Dataview v0.5.59
		5: Grandfather v1.0.1
		6: Window Collapse v1.0.2
		7: Achievements v0.0.10
		8: Inbox v3.0.0
		9: Calendar v1.5.10
		10: File Diff v1.1.0
		11: CSS Editor v0.5.0
		12: Canvas Send to Back v0.0.4
		13: Attachment Management v0.9.6
		14: CookLang Editor v0.3.0
		15: Settings Search v1.3.10
		16: Templater v2.2.3
		17: Linter v1.23.1
		18: Git v2.24.0
		19: BRAT v0.8.4

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Reply by huenchen** [reply]
Sorry for not answering any sooner. The problem is not related to Templater. In any file with or without templates the frontmatter cannot be changed. Thanks for your recommendations.
